moped-editor/src/views/staff/helpers.js

```diff
@@ -83,7 +84,7 @@ export const formatApiErrors = (errorsArray) =>
  */
 export const passwordLooksGood = (password) =>
   new RegExp(
-    "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{12,}$"
+    "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{12,}$'"
```

Comment:
I did a double-take on this but I'm pretty sure this added single quote needs to be removed. Above, the whole regex pattern in the returned error is wrapped in single quotes and the pattern used in the api is `^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{12,}$` (no single quote at the end)

Reply:
good catch, my faulty copy and paste job from above